capstone/src/test/java/com/google/sps/servlets/PostTest.java

```diff
@@ -0,0 +1,160 @@
+package com.google.sps.objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.Objects.Post;
+import com.google.sps.Objects.Comment;
+
+/**
+ * Unit tests for Post.
+ *
+ */
+public class PostTest {
+
+  private static final String AUTHOR_ID = "123123123";
+  private static final String POST_TEXT = "a great post";
+  private static final String CHALLENGE_NAME = "run 4 miles";
+  private static final String IMG = "";
+  private static final long TIMESTAMP = 123123123;
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setDefaultHighRepJobPolicyUnappliedJobPercentage(0));
+
+  private Post post;
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    post = 
+      new Post(
+        4324344, /* postId */ 
+        AUTHOR_ID, /* authorId */ 
+        POST_TEXT, /* postText */
+        new ArrayList<Comment>(), /* comments */
+        CHALLENGE_NAME, /* challengeName */
+        TIMESTAMP, /* timestamp */
+        IMG, /* img */
+        new HashSet<String>() /* likes */);
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+    post = null;
+  }
+
+  @Test
+  public void getPostTextTest() {
+    assertEquals(post.getPostText(), POST_TEXT);
+  }
+
+  @Test 
+  public void getImgtTest() {
+    assertEquals(post.getImg(), IMG);
+  }
+
+  @Test
+  public void getChallengeNameTest() {
+    assertEquals(post.getChallengeName(), CHALLENGE_NAME);
+  }
+
+  @Test 
+  public void getCommentsTest() {
+    assert post.getComments().size() == 0;
+  }
+
+  @Test
+  public void getLikesTest() {
+    assert post.getLikes().size() == 0;
+    post.addLike("user 1");
+    post.addLike("user 2");
+    HashSet<String> testLikes = new HashSet<String>();
+    testLikes.add("user 1");
+    testLikes.add("user 2");
+    
+    assert testLikes.containsAll(post.getLikes());
+  }
+
+  @Test
+  public void addCommentTest() {
+    Comment testComment1 =  
+      new Comment(
+        4324344, /* userId */ 
+        "a great comment", /* commentText */ 
+        "123123123" /* userId */ );
```

Comment:
```java
  /* userId */ 4324344,
  /* commentText */ "a great comment",
  /* userId */ "123123123"
```

reads a bit cleaner to me, because then you can read all the parameter names in a column.